### PR TITLE
Change SSLProtocol in apache::vhost to be space separated

### PR DESCRIPTION
**Pull request by bmfurtado**
https://tickets.puppetlabs.com/browse/MODULES-2655

### CODE REVIEW

File: templates/vhost/_ssl.erb [hunk 1]
```diff
@@ -26,7 +26,7 @@
   SSLProxyEngine On
   <%- end -%>
   <%- if @ssl_protocol -%>
-  SSLProtocol             <%= @ssl_protocol %>
+  SSLProtocol             <%= @ssl_protocol.compact.join(' ') %>
```
Comment (igalic) [inline templates/vhost/_ssl.erb]: i'd go as far as to make it
```ruby
[@ssl_protocol].flatten.compact.join(' ')
```

**Comment by HelenCampbell** [comment]
Hello @bmfurtado ,
Well spotted on this! Thank you for your work and @igalic for your input to help get the tests passing :)
It would be awesome if you could put a test in place to cover the array input and the readme could also be updated to mention that a single space separated string also works. After that I'll be happy to merge this in.

**Comment by bmfurtado** [comment]
@HelenCampbell I've added an acceptance test and updated the docs, is this ok?

**Comment by bmfurtado** [comment]
Disclaimer: For reasons I haven't had time to investigate I can't run the tests and it doesn't look like Travis runs acceptance tests (for some reason?) so I'm not really sure this works :)

**Comment by igalic** [comment]
Let's see :P

**Pull request merged**